patterns/cli/commands/delete.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+from typer import Option, Argument
+
+from patterns.cli.services.delete import delete_graph
+from patterns.cli.services.deploy import deploy_graph_version
+from patterns.cli.services.lookup import IdLookup
+from patterns.cli.services.output import sprint, abort_on_error
+
+_graph_help = "The location of the graph.yml file of the graph to delete"
+_graph_id_help = "The id of the graph version to delete"
+
+
+def delete(
+    graph_id: str = Option(""),
+    graph: Path = Argument(None, exists=True, help=_graph_help),
+):
+    """Delete a graph from the Patterns studio.
+
+    This will not delete any files locally.
+    """
```

Comment:
Would be nice if there was a confirmation prompt (unless a `-f` or `-y` flag is provided or something), since this is an irreversible and very destructive action (makes any data and state for that graph in the database inaccessible)

Reply:
good idea